### PR TITLE
Update Redcap parser

### PR DESCRIPTION
**Pull request by mpgxvii**
- Update Redcap parser to update range field on slider input types to fix issues with aRMT not processing fields correctly

### CODE REVIEW

File: utilities/RedcapParser.js [hunk 1]
```diff
@@ -116,6 +127,8 @@ function REDCapConverter(redcap_json) {
           rawContent[key].field_type = 'range-info';
         } else if (rawContent[key].field_annotation.includes('info-type')) {
           rawContent[key].field_type = 'info';
+        } else if (rawContent[key].field_note.includes('info-type')) {
```
Comment (yatharthranjan) [inline utilities/RedcapParser.js:130]: Is this in field note or field annotation like others?

File: utilities/RedcapParser.js [hunk 2]
```diff
@@ -154,6 +167,13 @@ function REDCapConverter(redcap_json) {
     return val;
   };
 
+  removeSpecialCharacters = function(val) {
+    if (val) {
+      return val.replace(/[^a-zA-Z0-9]/g, '');
```
Comment (yatharthranjan) [inline utilities/RedcapParser.js:172]: Should the code be alphanumeric or should it just be numeric?

File: utilities/RedcapParser.js (hunk 1 above)
Comment (mpgxvii) [reply utilities/RedcapParser.js:130]: Ah yes there were some questionnaires that had it in field_note as well, so I just added it.

File: utilities/RedcapParser.js (hunk 2 above)
Comment (mpgxvii) [reply utilities/RedcapParser.js:172]: It can be both I think. There were some existing questionnaires that have words as the code.

**Review by yatharthranjan — APPROVED** [review]
Thanks, LGTM.

**Pull request merged**